weed/storage/needle_map/compact_map_perf_test.go

```diff
@@ -90,3 +90,34 @@ func PrintMemUsage(totalRowCount uint64) {
 func bToMb(b uint64) uint64 {
 	return b / 1024 / 1024
 }
+
+func TestCompactSection_Get(t *testing.T) {
+	var maps []*CompactMap
+	totalRowCount := uint64(0)
+	indexFile, ie := os.OpenFile("../../1655451851227025229-502m-single-1_388.idx",
```

Comment:
where is this file? Seems this test does not work at all.

Reply:
[1655451851227025229-502m-single-1_388.idx.zip](https://github.com/seaweedfs/seaweedfs/files/9654785/1655451851227025229-502m-single-1_388.idx.zip)

Reply:
the file  is 38MB, so i didn't commit it

Reply:
try to create a test case directly. reproduce first, then fix.

Reply:
reproduce  pipline 
1. remove the compact_map.go Get function  line 169-171
2. run the test TestCompactSection_Get func ,  may test failed of  get error 
3. recovery the compact_map.go Get function  line 169-171  , run the TestCompactSection_Get test will pass